lms/templates/navigation.html

```diff
@@ -124,15 +124,21 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         % endif
       </%block>
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
```

Comment:
Ah ok thanks for the context and for flagging this. We have had Frances review the SASS changes here so if your review was for the rest of the PR that seems good to me. 

Here is the story I created with the details you described above. https://openedx.atlassian.net/browse/SOL-1043

Thanks!